site/_posts/2017-09-19-lwimiq-all-hands-busy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Last Week in ManageIQ: All Hands Busy"
+author: cybette
+date: 2017-09-19
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Last week, all engineering hands were busy in ManageIQ Engineering All-Hands meeting. A few stray hands still found the opportunity to submit some PRs, but there were no available fingers for typing up an entry for ["Last week in ManageIQ"][lwimiq tag]. So please allow me, [Carol][carol github], to paw in a few words in this edition.
+
+## Green thumbs
+
+As mentioned during [last Sprint][Sprint 68], [Greenkeeper.io][greenkeeper] was added to [Service UI repo][manageiq-ui-service] to keep dependencies updated a lot faster.
+
+If you take a look at the [PRs merged][manageiq-ui-service PRs merged] in this repo last week, you will see many of them with green 'greenkeeper' labels, as various dependencies were updated to their latest versions.
+
+## Iron fists
+
+When it comes to paying technical debt, sometimes one has got to be ruthless. A bunch of constants [were][UI PR 2138] [placed][UI PR 2139] [on][UI PR 2141] [the][UI PR 2142] [chopping][UI PR 2145] [board][UI PR 2158], decreed to be removed from UiConstants. As those constants found new homes in appropriate modules, the UiConstants module was also [permanently deleted][UI PR 2165].
```

Comment:
Possibly 'board' could be switched to 'block'.

Reply:
yep! I was probably hungry...